pkgs/development/python-modules/panflute/default.nix

```diff
@@ -0,0 +1,33 @@
+{ stdenv
+, lib
+, python3Packages
+, fetchPypi
+, click
+, pyyaml
+}:
+
+let
+  inherit (python3Packages) buildPythonPackage isPy3k;
```

Comment:
Since this is included from `pkgs/top-level/python-packages.nix`, it is already using a `callPackage` with the python scope. So you can just take these as inputs directly.